solutions/client-setup/setters.yaml

```diff
@@ -38,53 +38,77 @@ data:
   # General Settings Values
   ##########################
   #
+  # Use the same Google Cloud Organization ID that was used when deploying the core-landing-zone package
+  # customization: required
   org-id: "0000000000"
+  #
   # The environment (dev, preprod, prod), used in the client folder display name
+  # customization: required
   environment: env
   #
   ##########################
   # Management Project
   ##########################
   #
-  # This is the project where the config controller instance is running
-  # Values can be viewed in the Project Dashboard
+  # The management project is where the Landing Zone config controller instance is running.
+  # The $PROJECT_ID (management-project-id) is defined during Initial Organization Configuration (https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/landing-zone-v2/README.md#initial-organization-configuration)
+  # customization: required
   management-project-id: management-project-12345
+  #
+  # The management-project-number can be obtained from the Dashboard via https://console.cloud.google.com/home/dashboard?project=$PROJECT_ID
+  # Alternatively, obtain the management-project-number from gcloud: gcloud projects list --filter="${PROJECT_ID}" '--format=value(PROJECT_NUMBER)'
+  # customization: required
   management-project-number: "0000000000"
+  #
+  # kubernetes namespace set to the default, config-control. Do not change this value.
+  # customization: Do not change this value.
   management-namespace: config-control
   #
   ##########################
   # Client
   ##########################
   #
-  # Name for the client, lowercase only
+  # Name for the client, lowercase only.
+  # customization: required
   client-name: 'client1'
-  # Billing Account ID to associate with the client's managment project, the projects-sa will need billing user permission
-  # alternatively, it can be set to the core landing zone billing id, the client management project contains very limited resources
+  #
+  # Clients Billing Account ID to associate with the client's management project. The projects-sa will need billing user role permission granted.
```

Comment:
agree

Reply:
it may not always be true, this setter has a few different possible scenarios...

1. if the `client-billing-id` is the same as the core landing zone `billing-id`
    - the post-deployment step performed for `projects-sa` would be sufficient for this package
2.  if the `client-billing-id` is different than the core landing zone `billing-id`, and the client-billing-id is used
    - the post-deployment step performed for `projects-sa` would not be sufficient for this package, it would need to be granted billing user on the client's billing id
3. if the `client-billing-id` is different than the core landing zone `billing-id`, but the core-landing-zone billing-id is used to avoid granting additional permissions to the client billing (the alternative described on line 76)
    - the post-deployment step performed for `projects-sa` would be sufficient for this package